.github/workflows/ci.yml

```diff
@@ -55,17 +55,17 @@ jobs:
             libuv1-dev \
             libvterm-dev \
             locales \
-            lua-busted \
-            lua-check \
-            lua-filesystem \
-            lua-inspect \
-            lua-lpeg \
-            lua-luv-dev \
-            lua-nvim \
             luajit \
             ninja-build \
             pkg-config
 
+      - name: Install luarocks
+        run: |
+          sudo apt install -y lua5.1 luarocks
+          echo "using luarocks: "
+          luarocks path
+          sudo luarocks install luacheck
+          sudo luarocks --lua-version=5.1 build luv
```

Comment:
Why is this switching to luarocks instead of using the distributions packages?

Reply:
It's because lua-luv-dev is too old, you can see the test here: https://github.com/neovim/neovim/runs/7211684631?check_suite_focus=true

Reply:
That's why the lint job is still using [bundled LUV](https://github.com/neovim/neovim/blob/c68f1d7263cd5296e723d8cfee914c72f58138c3/.github/workflows/env.sh#L56-L58).

Reply:
doesn't that mean that this description is then obsolete?
https://github.com/neovim/neovim/blob/f0503371852ae8f142a4721e9b5670eee1ebb4f3/.github/workflows/ci.yml#L21-L24

or at least we should update this block, which otherwise looks identical to the one that install all the deps
https://github.com/neovim/neovim/blob/9ced05413474a7c8b8a8b2f36a27db29a37dfaf6/.github/workflows/ci.yml#L103-L104